tools/tsp-client/package.json

```
@@ -52,7 +52,7 @@
   "dependencies": {
     "@autorest/core": "^3.10.2",
     "@autorest/openapi-to-typespec": "0.10.5",
-    "@azure-tools/rest-api-diff": ">=0.1.0 <1.0.0",
+    "@azure-tools/rest-api-diff": "^0.1.8",
```

IIRC we had to explicitly specify the range so that minor version updates are allowed, since this package doesnt have a major version yet. If you want to specify a min version you could do:
```suggestion
    "@azure-tools/rest-api-diff": ">=0.1.8 <1.0.0",
```

---

Once the tool has a major version we should be able to simplify this

---

Yeah, I didn't actually manually change this file... Hmm.... I wonder if this command did.... Need to check that.